vimiv/imutils/filelist.py

```diff
@@ -122,7 +122,7 @@ def exif_date_time() -> str:
     data in the statusbar. If there are any requests/ideas for more, this can
     be used as basis to work with.
     """
-    return imutils.exif.exif_date_time(current())
+    return imutils.exif.ExifHandler(current()).exif_date_time()
```

Comment:
This could possibly be changed according to the comment above if we opt for the exception version.